cypress/TestCases/FireboltCertification/Accessibility.feature

```diff
@@ -138,13 +139,13 @@ Feature: Accessibility
             | Set windowColor-#7f7f7f            | windowColor       | #7F7F7F             |
             | Set windowOpacity-40               | windowOpacity     | 40                  |
 
-   @sdk @transport @requiresPlatformImplementation
-   Scenario: Accessibility.onClosedCaptionsSettingsChanged event - Positive Scenario: Clear listeners
-      Given '3rd party app' registers for the 'accessibility onClosedCaptionsSettingsChanged' event using the 'Firebolt' API
-      And 3rd party stops listening to the event 'accessibility onClosedCaptionsSettingsChanged event'
-      When 1st party app invokes the 'Firebolt' API to 'disable closedCaptions'
-      Then 'Firebolt' platform responds to '1st party app' for 'disable closedCaptions'
-      And 'Firebolt' platform does not trigger event for 'onclosedCaptionsSettingsChanged'
+    @sdk @transport @requiresPlatformImplementation
+    Scenario: Accessibility.onClosedCaptionsSettingsChanged event - Positive Scenario: Clear listeners
+        When '3rd party app' registers for the 'accessibility onClosedCaptionsSettingsChanged' event using the 'Firebolt' API
+        And 1st party stops listening to the event 'accessibility onClosedCaptionsSettingsChanged event'
+        And 1st party app invokes the 'Firebolt' API to 'disable closedCaptions'
+        Then 'Firebolt' platform responds to '1st party app' for 'disable closedCaptions'
+        And 'Firebolt' platform does not trigger event for 'onclosedCaptionsSettingsChanged'
```

Comment:
also where are the changes to modify sdk behavior test cases ? cypress/TestCases/FireboltCertification/SDKBehavior